### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by boroda4436**

### CODE REVIEW

File: src/main/java/mate/academy/bot/telegrambot/handlers/MySuperBot.java [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+import org.telegram.telegrambots.bots.TelegramLongPollingBot;
+import org.telegram.telegrambots.meta.api.methods.send.SendMessage;
+import org.telegram.telegrambots.meta.api.objects.Message;
+import org.telegram.telegrambots.meta.api.objects.Update;
+import org.telegram.telegrambots.meta.api.objects.replykeyboard.ReplyKeyboardMarkup;
+import org.telegram.telegrambots.meta.api.objects.replykeyboard.buttons.KeyboardRow;
+import org.telegram.telegrambots.meta.exceptions.TelegramApiException;
+
+public class MySuperBot extends TelegramLongPollingBot {
+    private static final String WHAT_THE_TIME_REQUEST = "What time is now?";
```
Comment (roman-mate) [inline src/main/java/mate/academy/bot/telegrambot/handlers/MySuperBot.java]:
```suggestion
    private static final String TIME_REQUEST = "What time is it now?";
```

File: src/main/java/mate/academy/bot/telegrambot/handlers/MySuperBot.java [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+    public static final String WHAT_THE_DATE_REQUEST = "What date is today?";
```
Comment (roman-mate) [inline src/main/java/mate/academy/bot/telegrambot/handlers/MySuperBot.java]:
```suggestion
    public static final String DATE_REQUEST = "What date is it today?";
```

File: src/main/java/mate/academy/bot/telegrambot/handlers/MySuperBot.java [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+    public static final String WHAT_IS_THE_CAPITAL_OF_GB_REQUEST = "What the city is the capital of GB?";
```
Comment (roman-mate) [inline src/main/java/mate/academy/bot/telegrambot/handlers/MySuperBot.java]:
```suggestion
    public static final String CAPITAL_OF_GB_REQUEST = "What is the capital of GB?";
```

File: src/test/java/mate/academy/bot/telegrambot/TelegramBotApplicationTests.java [hunk 4]
```diff
@@ -0,0 +1,13 @@
+package mate.academy.bot.telegrambot;
```
Comment (roman-mate) [inline src/test/java/mate/academy/bot/telegrambot/TelegramBotApplicationTests.java:1]: remove this file?

**Review by roman-mate — APPROVED** [review]

File: src/test/java/mate/academy/bot/telegrambot/TelegramBotApplicationTests.java (hunk 4 above)
Comment (boroda4436) [reply src/test/java/mate/academy/bot/telegrambot/TelegramBotApplicationTests.java:1]: this is standard SpringBoot smoke test that verifies the app is running OK

**Pull request merged**